src/features/nav/NavOrgs.tsx

```diff
@@ -107,18 +123,46 @@ export const NavOrgs = ({
   orgs,
   currentCircle,
   currentOrg,
+  onHideCurrentOrg,
 }: {
   orgs: NavOrg[];
   currentCircle: NavCircle | undefined;
   currentOrg: NavOrg | undefined;
+  onHideCurrentOrg: () => void;
 }) => {
   const [viewOrgList, setViewOrgList] = useState(false);
+  const storedHiddenOrgs = localStorage.getItem('hiddenOrgs');
+  const initialHiddenOrgs = storedHiddenOrgs
+    ? JSON.parse(storedHiddenOrgs)
+    : [];
+  const [hiddenOrgs, setHiddenOrgs] = useState<number[]>(initialHiddenOrgs);
+
+  const hideOrgsHandler = (orgId: number) => {
+    setHiddenOrgs(prevHiddenOrgs => [...prevHiddenOrgs, orgId]);
+  };
+
+  useEffect(() => {
+    localStorage.setItem('hiddenOrgs', JSON.stringify(hiddenOrgs));
+    if (hiddenOrgs.includes(currentOrg?.id)) onHideCurrentOrg();
```

Comment:
if I navigate directly to an org, via URL like http://localhost:3000/organizations/18 then I see the org, but I don't see it in the sidenav. So this whole pattern seems confusing.

Maybe you should not be able to hide current orgs, and only in the menu hide them. Also seems like maybe their should be a confirmation of some sort, its easy to click accidentally.
![image](https://github.com/coordinape/coordinape/assets/83605543/eac0e926-a993-42b3-bdf9-a4c81b933b76)